config/icons.js

```diff
@@ -44,6 +44,7 @@ module.exports = function () {
       "plus",
       "users",
       "question",
+      "magic",
```

Comment:
Did you try `wand-magic-sparkles` [fa](https://fontawesome.com/icons/wand-magic-sparkles?f=classic&s=solid) for more :sparkles: ? Or was it to noisy?